src/main/java/com/nionios/trial/api/ExpenditureEndpoint.java

```diff
@@ -34,6 +32,18 @@ public Response expendituresInProgress() {
         return Response.ok(expenditures).build();
     }
 
+
+    // change state
+    @PATCH
+    @Path("/changestate/{id}/state/{state}")
```

Comment:
Can you change this to "/{id}/state/{state}"?

Reply:
Done